db/schema.rb

```diff
@@ -139,7 +139,6 @@
     t.float    "max_value"
     t.float    "min_value"
     t.datetime "calibrated_at"
-    t.boolean  "smaxtec_sensor"
```

Comment:
Is the removal of the `smaxtec_sensor` property intended? Not sure how this is related to issue #373…